web/src/app/schedules/temp-sched/TempSchedTimesStep.tsx

```diff
@@ -7,24 +7,33 @@ import {
 } from '@material-ui/core'
 import { FormField } from '../../forms'
 import { ISODateTimePicker } from '../../util/ISOPickers'
-import { contentText, StepContainer } from './sharedUtils'
+import { contentText, StepContainer, Value } from './sharedUtils'
 import { ScheduleTZFilter } from '../ScheduleTZFilter'
 
+import { DateTime } from 'luxon'
+
 const useStyles = makeStyles({
   contentText,
 })
 
 type TempSchedTimesStepProps = {
   scheduleID: string
   stepText: string
+  value: Value
 }
 
 export default function TempSchedTimesStep({
   scheduleID,
   stepText,
+  value,
 }: TempSchedTimesStepProps): JSX.Element {
   const classes = useStyles()
 
+  function isValid(): Error | null {
+    if (DateTime.fromISO(value.start) < DateTime.fromISO(value.end)) return null
```

Comment:
Since this returns an error not a bool call it `validate` -- `is` implies a boolean value.

For the comparison we can use `isBefore` now that it's in a helper lib